cache.go

```diff
@@ -334,7 +353,7 @@ func (nc *NumberCache[K, V]) Increment(key K, n V) V {
 	defer nc.nmu.Unlock()
 	got, _ := nc.Cache.Get(key)
 	nv := got + n
-	nc.Cache.Set(key, nv)
+	nc.Cache.Set(key, nv, WithKeepTTL())
```

Comment:
We do not intend to make destructive changes to this library to ensure backward compatibility unless there is a good reason to do so. Why is this change necessary? Please provide a clear reason.

Reply:
The default rules for redis/memcached and more cache libraries... We do not recommend using different designs to avoid thinking burdens